app/models/suggestion.py

```diff
@@ -0,0 +1,79 @@
+from .. import db
+
+class Suggestion(db.Model):
+	"""
+	Association between a resource and potential suggestions for it
+	"""
+	__tablename__ = 'suggestions'
+	id = db.Column(db.Integer, primary_key = True)
+	#suggestion_type is 0 for insertion, 1 for edit, and 2 for deletion
+	suggestion_type = db.Column(db.Integer)    
```

Comment:
do we need this field? Won't admin's actions be the same regardless of the type (specifically, take a look at the suggestion and then making necessary changes)?